01-exercise-easy/05-nested-List/nestedLists_Dev.py

```diff
@@ -0,0 +1,17 @@
+if __name__ == '__main__':
+    records = []
+    segmaior = []
+    for _ in range(int(input())):
+        name = input()
+        score = float(input())
+        records.append([score, name])
+    records.sort()
+    for alunos in records:
```

Comment:
Em vez de fazer um for dentro de outro for, pode ser solucionado da seguinte maneira
```suggestion
     unique_scores = sorted({score for name, score in records})
     segunda_menor = unique_scores[1]
     second_lowest_students = [name for name, score in records if score == segunda_menor]
     second_lowest_students.sort()
```
- Ordena as notas e remove duplicatas
- A segunda menor nota é o segundo elemento da lista ordenada
- Filtra os alunos que têm a segunda menor nota
- Ordena os nomes dos alunos alfabeticamente